test/user.model.test.js

```diff
@@ -148,7 +148,7 @@ describe('user.models.test.js', () => {
       const decoded = auth.verifyJwt(token);
 
       // And then the decoded token should contain the user data
-      expect(decoded.sub).to.equal(user.id);
+      expect(+decoded.sub).to.equal(user.id);
```

Comment:
What was the value and type of `decoded.sub` before the operation?

Reply:
It was a string.